src/table/TableWrapper.jsx

```diff
@@ -73,10 +75,44 @@ export default function TableWrapper(props) {
     };
   }, []);
 
+  useEffect(() => {
+    if (focusedCellCoordsState[0] < 2) {
+      tableSection.current.scrollTo({
+        top: 0,
+        behavior: 'smooth',
+      });
+    } else if (isMovingTop) {
+      setIsMovingTop(false);
+
+      const rowElements = rootElement.getElementsByClassName('sn-table-row');
+      const [x, y] = focusedCellCoordsState;
+      const cell = rowElements[x]?.getElementsByClassName('sn-table-cell')[y];
+
+      if (cell.offsetTop - cell.offsetHeight * 2 < tableSection.current.scrollTop) {
+        tableSection.current.scrollTo({
+          top: tableSection.current.scrollTop - cell.offsetHeight || 0,
+          behavior: 'smooth',
+        });
+      }
+    }
+  }, [tableSection, focusedCellCoordsState, isMovingTop, rootElement]);
+
+  const handleFocusedCellCordsUpd = useCallback((newCoords) => {
+    setFocusedCellCoordsState(newCoords);
+  }, []);
+
+  const handleSetMovingTop = useCallback((val) => setIsMovingTop(val), []);
```

Comment:
Do these make any difference, compared to passing just the setState?

Reply:
you are right, in our case it does not makes any difference if we pass the state setter by itself. 

i'll fix it asap 🙏